### ISSUE TITLE
[systemd integration] Need to ensure that systemd timer is automatically enabled

### ISSUE THREAD

**Issue by rtamalin** [SUSE/scc-hypervisor-collector#33]
After installing the latest packages after adding the systemd service and timer, and rebooting, I don't see the timer service being enabled:

```
sudo systemctl status scc-hypervisor-collector.{timer,service}
\u25cf scc-hypervisor-collector.timer - update SCC daily with collecter hypervisor details
     Loaded: loaded (/usr/lib/systemd/system/scc-hypervisor-collector.timer; static)
     Active: inactive (dead)
    Trigger: n/a
   Triggers: \u25cf scc-hypervisor-collector.service
       Docs: man:scc-hypervisor-collector(1)
             man:scc-hypervisor-collector.service(8)

\u25cf scc-hypervisor-collector.service - run scc-hypervisor-collector if valid config defined
     Loaded: loaded (/usr/lib/systemd/system/scc-hypervisor-collector.service; static)
     Active: inactive (dead)
       Docs: man:scc-hypervisor-collector(1)
             man:scc-hypervisor-collector.service(8)

```

This needs investigation & fixing.

**Comment by rtamalin** [comment]
The systemd .timer script is missing an `[Install]` section `WantedBy=` setting.

**Comment by rtamalin** [comment]
This was fixed by #39
